.github/workflows/build-ubuntu.yml

```diff
@@ -24,19 +24,15 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-18.04]
-        client-gg: ['ON', 'OFF']
```

Comment:
Can we leave it `['ON', 'OFF']` and remove `include:` section?

Reply:
I've changed variants we use.